File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int  i = 0;
        int  j = 1;
        int n = nums.length;
        while (j<n){
            if(nums[i]!=nums[j]){
                i+=1;
                nums[i] = nums[j];
            }
            j+=1;
        }
        return i+1;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        set_nums = set()
        
        for i in range(len(nums)):
            set_nums.add(nums[i])
        k = len(set_nums)

        for i in range(len(set_nums)):
            nums[i] = set_nums.pop()

        return k